frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -20,12 +20,15 @@ export interface ITableInstanceProps {
   showGuiIcon: boolean;
   extended: boolean;
   showAdvanced?: boolean;
+  showCheckbox?: boolean;
   handleSorting?: (sortingType: string, sorting: number) => void;
   handleManagerSorting?: (
     sortingType: string,
     sorting: number,
     sortingTemplate: string
   ) => void;
+  selectiveDestroy?: string[];
+  selectToDestroy?: (instanceId: string) => void;
```

Comment:
Does this have to be optional?

Reply:
Yes, unfortunately because the dashboard also use the `TableInstance` without the selective destroy